src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/AsyncHelpers.CoreCLR.cs

```diff
@@ -93,6 +102,29 @@ internal sealed unsafe class Continuation
         //
         public byte[]? Data;
         public object?[]? GCData;
+
+        public object GetContinuationContext()
+        {
+            int index = 0;
```

Comment:
Could this (and similar) index calculations be simplified to `int index = BitOperations.PopCount((uint)(Flags & (CorInfoContinuationFlags.CORINFO_CONTINUATION_RESULT_IN_GCDATA | CorInfoContinuationFlags.CORINFO_CONTINUATION_NEEDS_EXCEPTION)))`?

Reply:
Doesn't look simpler to me, but yes it could be optimized slightly like that.